.env.example

```
@@ -163,6 +163,7 @@ DEMO_VIDEO_URL=
 NEXT_PUBLIC_TURNSTILE_SITE_KEY=
 TURNSTILE_SECRET_KEY=
 # Production synthetic signup canary. Configure only in Doppler prd.
+E2E_SYNTHETIC_MODE=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The E2E_SYNTHETIC_MODE key should go before the NEXT_PUBLIC_BUILD_SHA key

(UnorderedKey)